bgpd/bgp_nht.c

```diff
@@ -745,8 +745,17 @@ static void evaluate_paths(struct bgp_nexthop_cache *bnc)
 			bnc_is_valid_nexthop =
 				bgp_isvalid_labeled_nexthop(bnc) ? 1 : 0;
 		} else {
-			bnc_is_valid_nexthop =
-				bgp_isvalid_nexthop(bnc) ? 1 : 0;
+			if (bgp_update_martian_nexthop(
+				    bnc->bgp, afi, safi, path->type,
+				    path->sub_type, path->attr, rn)) {
+				if (BGP_DEBUG(nht, NHT))
+					zlog_debug(
+						"%s: Prefix %pRN (vrf %s), ignoring path due to martian or self-next-hop",
```

Comment:
Sorry for my pedantic request changes, I hate to say that, but please `Prefix` -> `prefix`. Just to be the same format as below. And we should remove {} (curls) for short if statements.

Reply:
Thanks, @ton31337. I've made the suggested changes.
I've removed the curl braces for the ELSE statement.
If we remove the braces for the first IF (if (bgp_update_martian_nexthop)) that would lead to an ambiguous ELSE statement. Hence, we need to retain it.
Kindly have a look.